lib/pre2017.js

```diff
@@ -0,0 +1,109 @@
+const fetch = require('node-fetch')
+const cheerio = require('cheerio')
+const https = require('https')
+
+const PRE_URL = [
+  'https://codein.withgoogle.com/archive/2016',
+  'https://codein.withgoogle.com/archive/2015',
+  'https://www.google-melange.com/archive/gci/2014', // Ignore InvalidLinkBear
+  'https://www.google-melange.com/archive/gci/2013', // Ignore InvalidLinkBear
+  'https://www.google-melange.com/archive/gci/2012', // Ignore InvalidLinkBear
+  'https://www.google-melange.com/archive/gci/2011', // Ignore InvalidLinkBear
+  'https://www.google-melange.com/archive/gci/2010', // Ignore InvalidLinkBear
```

Comment:
Modify `.coafile` to use `link_ignore_list`

ref https://github.com/coala/projects/blob/master/.coafile#L6